Makefile

```diff
@@ -24,7 +25,7 @@ lint-go:
 .PHONY: lint-proto
 lint-proto:
 	go mod vendor
-	buf lint api
+	buf lint api --error-format=$(BUF_LINT_ERROR_FORMAT)
```

Comment:
I see what you did. I honestly had no idea that the Action was using a different/specialized error format!